File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {

      set<int> st;

     for( int i=0; i<nums.size(); i++ ) {
        st.insert(nums[i]);
     }
       vector<int> ans(st.begin() , st.end());
       for( int i=0; i<ans.size(); i++ ) {
         cout << ans[i] <<" ";
       }
        nums = ans;
        return nums.size();
    }
};

File: Data Structures & Algorithms/monotonic-array/submission-0.java
class Solution {
    public boolean isMonotonic(int[] nums) {
        
        int i=0;
         if( nums.length > 1 ) {

            if( nums[0] <= nums[1] ) {

                while( i < nums.length-1 ) {
                    if( nums[i] > nums[i+1] ) {
                        return false;
                    }
                    i++;
                }
            }


            else {

                 while( i < nums.length-1 ) {
                    if( nums[i] < nums[i+1] ) {
                        return false;
                    }
                    i++;
                }


            }
         }
        

         return true;
    }
}